src/ripple/app/misc/AmendmentTable.h

```diff
@@ -272,22 +272,25 @@ class AmendmentTable
         // Inject appropriate pseudo-transactions
         for (auto const& it : actions)
         {
-            STTx trans (ttAMENDMENT);
-            trans.setAccountID (sfAccount, AccountID());
-            trans.setFieldH256 (sfAmendment, it.first);
-            trans.setFieldU32 (sfLedgerSequence, lastClosedLedger->seq() + 1);
+            STTx amendTx (ttAMENDMENT,
+                [&it, seq = lastClosedLedger->seq() + 1](STObject& obj)
```

Comment:
@HowardHinnant My thinking is that this particular use of a lambda is roughly equivalent to just opening a new scope with curly braces